cradle-core/src/main/java/com/exactpro/cradle/serialization/LegacyEventDeserializer.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2021-2022 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exactpro.cradle.serialization;
+
+import com.exactpro.cradle.testevents.BatchedStoredTestEvent;
+import org.apache.commons.lang3.SerializationUtils;
+
+import java.io.ByteArrayInputStream;
+import java.io.DataInputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+
+public class LegacyEventDeserializer {
```

Comment:
We need backward compatibility? @Vladimir-Panarin-Exactpro

Reply:
Backward compatibility is removed as discussed